routes/users.js

```diff
@@ -4,8 +4,8 @@ const bcrypt = require('bcryptjs');
 const passport = require('passport');
 
 //Bring in User Model
-let UserModel = require('../models/user');
-const dentalOffice = require('../models/dentalOffice');
+let userModel= require('../models/user');
```

Comment:
userModel is a bad name, check this out, cool programming stuff: https://www-freecodecamp-org.cdn.ampproject.org/c/s/www.freecodecamp.org/news/clean-coding-for-beginners/amp/